src/abstract/PeripheryPayments.sol

```diff
@@ -0,0 +1,67 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity >=0.7.5;
+
+import "@openzeppelin/token/ERC20/IERC20.sol";
+
+import "src/interfaces/IPeripheryPayments.sol";
+import "src/interfaces/IWETH.sol";
+
+import "src/libraries/TransferHelper.sol";
+
+import "src/abstract/PeripheryImmutableState.sol";
+
+abstract contract PeripheryPayments is IPeripheryPayments, PeripheryImmutableState {
+    receive() external payable {
+        require(msg.sender == address(WETH), "Not WETH");
+    }
+
+    /// @inheritdoc IPeripheryPayments
+    function unwrapWETH(uint256 amountMinimum, address recipient) public payable override {
+        uint256 balanceWETH = IWETH(WETH).balanceOf(address(this));
+        require(balanceWETH >= amountMinimum, "Insufficient WETH");
```

Comment:
non-standard error messages in this file + variable style

Reply:
fixed all imported files and interfaces, except `TransferHelper`